UPGRADING.rst

```diff
@@ -10,6 +10,13 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register indexer service accounts with SAM
+================================================
+
+Service accounts are now registered with SAM upon deployment.
+They can also manually be registered via ``make -C terraform sam``.
+However, the accounts must still be manually authorized for steward access by
```

Comment:
```suggestion
However, the account must still be authorized for repository read access (datasets and snapshot).
This is currently done manually by someone on the Azul team. Ask on Slack to find out more.
```

Reply:
Do datasets and snapshots have separate access permissions? If so, shouldn't both be tested?